client/lib/reader-site-store/index.js

```diff
@@ -30,6 +30,10 @@ function setSite( attributes ) {
 		attributes.state = State.COMPLETE;
 	}
 
+	if ( attributes.meta && attributes.meta.links ) {
```

Comment:
Good idea; switched to using `has` (https://github.com/Automattic/wp-calypso/commit/e099e0ad9e2b18ebc43d9031059d9b2df5929055)